scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -250,7 +251,7 @@
 
 (defn container-status->failure-reason
   "Maps kubernetes failure reasons to cook failure reasons"
-  [{:keys [name]} instance-id ^V1PodStatus pod-status ^V1ContainerStatus container-status]
+  [{:keys [name]} instance-id ^V1PodStatus pod-status ^V1Pod pod ^V1ContainerStatus container-status]
```

Comment:
Can strip out the `instance-id` argument and extract it from the `pod` argument

Reply:
Done